offsite_storage/settings.py

```diff
@@ -11,6 +11,8 @@
     settings, 'AWS_MEDIA_SECRET_ACCESS_KEY', AWS_SECRET_ACCESS_KEY)
 AWS_MEDIA_BUCKET_NAME = getattr(settings, 'AWS_MEDIA_BUCKET_NAME')
 
+AWS_S3_ENDPOINT_URL = getattr(
+    settings, 'AWS_S3_ENDPOINT_URL', 's3.amazonaws.com')
 AWS_HOST_URL = 'https://%(bucket_name)s.s3.amazonaws.com/'
```

Comment:
`AWS_HOST_URL` in this package is used only to determine static file url. So we shouldn't use direct region-based url. When we do, we'll end up with urls like `http://my-assets.s3-eu-west-1.amazonaws.com/style[hash].css` instead of nice `http://my-assets.s3.amazonaws.com/style[hash].css`